src/runtime/client-config-manager.ts

```diff
@@ -128,19 +112,17 @@ export class ClientConfigManager {
    * client options, rather than the default of letting the iframes decide the
    * display language. Note that this will return false if the lang option is
    * not set, even if forceLangInIframes was set.
-   * @return {boolean}
    */
-  shouldForceLangInIframes() {
+  shouldForceLangInIframes(): boolean {
     return (
       !!this.clientOptions_.forceLangInIframes && !!this.clientOptions_.lang
     );
   }
 
   /**
    * Determines whether a subscription or contribution button should be disabled.
-   * @returns {!Promise<boolean|undefined>}
    */
-  async shouldEnableButton() {
+  async shouldEnableButton(): Promise<boolean | void> {
```

Comment:
nit: is `Promise<void>` the replacement for `Promise<undefined>`?

Reply:
Yep, exactly